cromwell-drs-localizer/src/main/scala/drs/localizer/DrsLocalizerMain.scala

```diff
@@ -51,8 +56,24 @@ object DrsLocalizerMain extends IOApp with StrictLogging {
   }
 
   def runLocalizer(commandLineArguments: CommandLineArguments, accessTokenStrategy: AccessTokenStrategy): IO[ExitCode] = {
-    val drsObject = commandLineArguments.drsObject.get
-    val containerPath = commandLineArguments.containerPath.get
+    commandLineArguments.manifestPath match {
+      case Some(manifestPath) =>
+        val manifestFile = new File(manifestPath)
+        val csvParser = CSVParser.parse(manifestFile, Charset.defaultCharset(), CSVFormat.DEFAULT)
+        val exitCodes: IO[List[ExitCode]] = csvParser.asScala.map(record => {
+          val drsObject = record.get(0)
+          val containerPath = record.get(1)
```

Comment:
If the manifest file doesn't have the format we expect, what will happen here? Should we check the length of `record` and throw an exception?